modules/apps/commerce/commerce-order-rule-web/src/main/resources/META-INF/resources/cor_entry/details.jsp

```diff
@@ -47,7 +47,7 @@ String type = BeanParamUtil.getString(corEntry, renderRequest, "type", COREntryC
 					</div>
 
 					<div class="col-auto">
-						<aui:select label="type" name="type" required="<%= true %>">
+						<aui:select disabled="<%= true %>" label="type" name="type" required="<%= true %>">
```

Comment:
Since this page is only used to edit the `OrderRule` and the `type` can't be changed, does it even make sense to have a select here? I don't think so.